### PR TITLE
Fix/k0s runtime

### PR DESCRIPTION
**Pull request by starComingup**
### Describe what this PR does / why we need it
fix k0s cluster install bug, which because k0s service not loaded, and define external CRI const mistake.

### Does this pull request fix one issue?
NONE

### CODE REVIEW

File: pkg/runtime/k0s/utils.go [hunk 1]
```diff
@@ -29,6 +31,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+const DelayTimes = 5
```
Comment (allencloud) [inline pkg/runtime/k0s/utils.go:34]: You should add comment for this const value. @starComingup

File: pkg/runtime/k0s/utils.go [hunk 2]
```diff
@@ -74,3 +78,22 @@ func FetchKubeconfigAndGetKubectl(ssh ssh.Interface, host net.IP, rootfs string)
...
+		time.Sleep(time.Second * 2)
+		bytes, err := ssh.Cmd(host, "k0s status")
+		if err != nil {
+			return err
+		}
+		if strings.Contains(string(bytes), "Process") {
+			return nil
+		}
+	}
+	return errors.New("failed to start k0s")
```
Comment (allencloud) [inline pkg/runtime/k0s/utils.go:98]: Please make the error more readable. 
`failed to start k0s: failed to get k0s status after 10 seconds` ?

File: pkg/runtime/k0s/utils.go [hunk 3]
```diff
@@ -74,3 +78,22 @@ func FetchKubeconfigAndGetKubectl(ssh ssh.Interface, host net.IP, rootfs string)
...
+		times--
+		if times == 0 {
+			break
+		}
+		time.Sleep(time.Second * 2)
+		bytes, err := ssh.Cmd(host, "k0s status")
+		if err != nil {
+			return err
+		}
+		if strings.Contains(string(bytes), "Process") {
```
Comment (allencloud) [inline pkg/runtime/k0s/utils.go:94]: Add more comment that why containing `Process` means success.

File: pkg/runtime/k0s/utils.go (hunk 2 above)
Comment (starComingup) [reply pkg/runtime/k0s/utils.go:98]: sounds good, Thx!

**Review by allencloud — APPROVED** [review]

**Pull request merged**